src/Acuminator/Acuminator.Vsix/ServiceLocation/MefServiceLocator.cs

```diff
@@ -1,37 +1,37 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.Composition.Hosting;
-using System.Composition;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using CommonServiceLocator;
+﻿//using System;
```

Comment:
Please delete file as it is not used

Reply:
We decided to keep service locator-related entities because we're not sure what happened to this API in 2019, and we want to review in in the future.

Reply:
Ok. Do we have an item for this task?

Reply:
No, I’ll create a task

Reply:
I won't create a ticket for it. Instead I'll remove both files as I have written in the previous conversation

Reply:
Done